tutorials/hist/histv7/hist002_RHist_weighted.C

```diff
@@ -0,0 +1,117 @@
+/// \file
+/// \ingroup tutorial_histv7
+///
+/// Weighted filling of RHist and RBinWithError bin content type.
+///
+/// \macro_code
```

Comment:
Let's add the output to the doxygen-generated page?
[Example](https://root.cern/doc/master/rf101__basics_8C.html) 
```suggestion
/// \macro_code
/// \macro_output
```